Core/Transaction.cs

```diff
@@ -29,7 +29,7 @@ public Transaction(string cardNumber, Date date, Money amount, string descriptio
 
 		public new string ToString()
 		{
-			return $"{Amount.Amount} {Amount.Currency}: {Description}";
+			return $"{Date.ToLongString()}:" + $"{Amount.Amount}".PadLeft(10) +$" {Amount.Currency}  {Description}";
```

Comment:
Codacy found an issue: [Assign this magic number '10' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5872129)